fugue-deploy-aws/src/main/java/com/symphony/oss/fugue/aws/deploy/UploadOutputStream.java

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
copy the Apache comment block from another class in oss-fugue and replace this one